app.rb

```diff
@@ -2,13 +2,31 @@
 require './teacher'
 require './student'
 require './rental'
+require 'json'
+require 'fileutils'
+
+# rubocop:disable Metrics
 class App
   attr_accessor :people, :rentals, :books
 
   def initialize
     @rentals = []
     @books = []
     @people = []
+    @people_path = './../file/people.json'
+    @books_path = './../file/books.json'
+    @rentals_path = './../file/rentals.json'
```

Comment:
- [ ] Please ensure that the file folder is inside the main project directory, right now the file is being created among my personal projects.